<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>341</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Destination IP</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="ipLineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>port</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="portLineEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Message</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="messageLineEdit"/>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="sendTcpButton">
       <property name="text">
        <string>tcpsend</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="sendUdpButton">
       <property name="text">
        <string>udp send</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>50</y>
      <width>261</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 38pt &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string>TCP/UDP</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="outputTextEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>450</y>
      <width>951</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>150</y>
      <width>438</width>
      <height>58</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="1">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Listen on IP</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2" colspan="2">
      <widget class="QLineEdit" name="ListeningIp"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>TCP Port</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="TCPPortInput"/>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="UDPPort">
       <property name="text">
        <string>UDP Port</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="UDPPortInput"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>210</y>
      <width>481</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="1">
      <widget class="QPushButton" name="startUDPListenerButton">
       <property name="text">
        <string>startUDPListen</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="startTCPListenerButton">
       <property name="text">
        <string>startTCPListen</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QTextEdit" name="TCPTextEdit"/>
     </item>
     <item row="0" column="1">
      <widget class="QTextEdit" name="UDPTextEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="StopListeningTCP">
       <property name="text">
        <string>StopListening</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="StopListeningUDP">
       <property name="text">
        <string>StopListening</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
